ufersa/plataforma-sabia
# New login page

## Summary

Resolves #1092 

## Relevant technical choices



## QA Steps

1. Access `/entrar` page
2. Assert you can login
3. Assert inline login is shown when opening technology costs page
4. Assert you're redirected to login with a callback url when trying to access new technology page

## Checklist
